_deployment/sign.sh

```diff
@@ -0,0 +1,6 @@
+#!/usr/bin/env bash
```

Comment:
@phallobst the code depends on Bash to work. Many sh versions don't support wildcards.

Reply:
:confused: In the last 25 years of hacking shell scripts, I have never seen one that didn't support wildcards. I mean, we are not talking about recursive globbing like `**` in `zsh` or something...

Reply:
Busybox ash is one example of a system shell not supporting wildcards.

Reply:
```
BusyBox v1.30.1 (Ubuntu 1:1.30.1-4ubuntu6.3) built-in shell (ash)
Enter 'help' for a list of built-in commands.

/tmp $ touch a1 a2
/tmp $ for i in a*; do echo $i; done
a1
a2
/tmp $ mkdir b
/tmp $ mkdir b/c
/tmp $ echo ?/?
b/c
```
Are you running `busybox` on a toaster or something? :laughing:

Reply:
```sh
#!/bin/sh

for file in api/v*/*.json; do
  echo "$file"
done
```
outputs
```
api/v*/*.json
```

Reply:
That's because there is no matching file, not because the shell wouldn't support wildcards. The `bash` shows the same behaviour, unless you set the `nullglob` option there.

Reply:
🤦‍♂️ my bad.
How about `#!/usr/bin/env sh`?

Reply:
I don't really see the benefit, actually.

Reply:
Okay but can you see any downsides to having it? 🤔

Reply:
Other than "nobody in their right mind would do it like this"? No... :stuck_out_tongue_winking_eye: